packages/docusaurus-theme-classic/src/theme/Navbar/styles.css

```diff
@@ -26,6 +26,24 @@
   -webkit-tap-highlight-color: transparent;
 }
 
+.toggle {
+  position: relative;
+  display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
Indentation is off. Sort alphabetically (it's easier to binary scan the values to check for presence of a certain value).

We need to add a CSS style guide/linting soon 😓

Reply:
Sorted 🐛